spec/charts/count_chart/count_chart_spec.rb

```diff
@@ -42,16 +42,16 @@
     let(:chart) { Charts::CountChart.new [1] }
     it 'has a default item-colors' do
       expect(chart.colors).to eq([
-                                   '#e41a1d',
-                                   '#377eb9',
-                                   '#4daf4b',
-                                   '#984ea4',
-                                   '#ff7f01',
-                                   '#ffff34',
-                                   '#a65629',
-                                   '#f781c0',
-                                   '#888888'
-                                 ])
+        '#e41a1d',
+        '#377eb9',
+        '#4daf4b',
+        '#984ea4',
+        '#ff7f01',
+        '#ffff34',
+        '#a65629',
+        '#f781c0',
+        '#888888'
+      ])
```

Comment:
close the ending square bracket, and give the ending parenthesis on the next line.